src/kOS/Binding/BindingsUniverse.cs

```diff
@@ -56,13 +56,28 @@ public override void AddTo(SharedObjects shared)
                     FlightDriver.StartAndFocusVessel(game, game.flightState.activeVesselIdx);
                 });
 
-            shared.BindingMgr.AddGetter("LOADDISTANCE", () => Vessel.loadDistance);
+            shared.BindingMgr.AddGetter("LOADDISTANCE", () => PhysicsGlobals.Instance.VesselRangesDefault.orbit.load);
             shared.BindingMgr.AddSetter("LOADDISTANCE", val =>
             {
                 var distance = Convert.ToSingle(val);
-                Vessel.loadDistance = distance;
-                Vessel.unloadDistance = distance - 250;
+                PhysicsGlobals.Instance.VesselRangesDefault.landed.load = distance;
```

Comment:
This is simply here to keep our API the same for compatibility. we should revisit it to allow for manipulation of each type.